Adyen/UI/View Controllers/ADYViewController.swift

```diff
@@ -4,32 +4,39 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
-import Adyen
 import UIKit
 
-internal final class QRCodeViewController: UIViewController {
+/// :nodoc:
+/// `ADYViewController` serves as a height-aware `UIViewController`
+final public class ADYViewController: UIViewController {
     
-    private let qrCodeView: QRCodeView
+    /// :nodoc:
+    private let _view: UIView
```

Comment:
we have private vars with a `_` prefix, but only for `_isDropin` because it has to be public but yet used only internally so the `_` has a point, but here its not really the swift way I would say.

Reply:
`contentView` ?